source/module_relax/relax_old/lattice_change_cg.cpp

```diff
@@ -80,11 +80,20 @@ void Lattice_Change_CG::start(UnitCell &ucell, const ModuleBase::matrix &stress_
     static double xa, xb, xc, xpt, steplength, fmax; // the steepest descent method
     static int nbrent;
 
-    double *lat = new double[dim];
```

Comment:
I guess the 'new' has corresponding 'delete' code? You may need to modify them together.